scripts/datastore_upgrade.py

```diff
@@ -0,0 +1,330 @@
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../lib'))
+import appscale_info
+from constants import CONTROLLER_SERVICE
+from constants import APPSCALE_HOME
+import monit_interface
+import monit_app_configuration
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../AppDB'))
+import appscale_datastore_batch
+import datastore_server
+from dbconstants import *
+from zkappscale import zktransaction as zk
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../AppServer"))
+from google.appengine.datastore import entity_pb
+
+# The location on the local filesystem where Cassandra is installed.
+CASSANDRA_DIR = "/opt/cassandra"
+
+# The location of the Cassandra binary on the local filesystem.
+CASSANDRA_EXECUTABLE = CASSANDRA_DIR + "/cassandra/bin/cassandra"
+
+PID_FILE = "/var/appscale/appscale-cassandra.pid"
+
+# The default port to connect to Cassandra.
+CASSANDRA_PORT = 9999
+
+# The default port to connect to ZooKeeper.
+ZOOKEEPER_PORT = 2181
```

Comment:
You can use the constant defined in AppDB/zkappscale/zktranscation.py instead of creating a new one.